src/main/java/org/reactome/release/qa/check/NewRegulationChecker.java

```diff
@@ -0,0 +1,120 @@
+        }
+        return report;
+    }
+
+    /**
+     * This method checks to see if a new regulation instance has been added to the ReactionlikeEvent by comparing between slices.
+     * If it does, it then compares the number of reviewed instances between slices. If the number of regulation instances is greater AND
+     * the number of reviewed instances is the same, it is flagged to indicate that this ReactionlikeEvent still needs to reviewed before
+     * it can be released.
+     * @param currentRlE GKInstance -- ReactionlikeEvent from current slice.
+     * @param previousRlE GKInstance -- Same ReactionlikeEvent, but from previous slice.
+     * @return boolean -- True if a new, unreviewed regulation instance has been added.
+     * @throws Exception -- DBA exceptions.
+     */
+    private boolean changedRegulatedByWithoutNewReviewed(GKInstance currentRlE, GKInstance previousRlE) throws Exception {
+
+        // If previousRlE does not exist, we ignore it since that does not pertain to this particular QA check.
+        if (previousRlE == null) {
+            return false;
+        }
+        // Compare contents of the 'regulatedBy' attribute of the current and previous versions of the RlE.
+        boolean sameRegulatedBy = isSameRegulatedByValues(
+                currentRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy),
+                previousRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy)
+        );
+
+        int currentRlEReviewed = currentRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+        int previousRlEReviewed = previousRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+
+        // The actual QA check
+        return !sameRegulatedBy && currentRlEReviewed == previousRlEReviewed;
+    }
+
+    /**
+     * Takes the two 'regulatedBy' lists from the current and previous versions of a ReactionlikeEvent, and
+     * compares their contents, returning false if they differ.
+     * @param regulatedByInstancesCurrent List<GKInstance> -- Contents of currentRlE's 'regulatedBy' attribute
+     * @param regulatedByInstancesPrevious List<GKInstance> -- Contents of previousRlE's 'regulatedBy' attribute
+     * @return boolean -- true if lists are equal, false if not.
+     */
+    private boolean isSameRegulatedByValues(List<GKInstance> regulatedByInstancesCurrent, List<GKInstance> regulatedByInstancesPrevious) {
+        if (regulatedByInstancesCurrent.size() != regulatedByInstancesPrevious.size()) {
+            return false;
+        }
+        Set<Long> list1DbIds = getInstanceListDBIDs(regulatedByInstancesCurrent);
+        Set<Long> list2DbIds = getInstanceListDBIDs(regulatedByInstancesPrevious);
+        return list1DbIds.equals(list2DbIds);
+    }
+
+    // Return all unique DBIDs from instances in the list
+    private Set<Long> getInstanceListDBIDs(List<GKInstance> list) {
+        return list.stream().map(GKInstance::getDBID).collect(toSet());
```

Comment:
Thought the set comparison should work, however, better to use a sorted list for defensive programming.